src/test/java/com/scottlogic/pod/spark/playground/steps/WordCountSteps.java

```diff
@@ -0,0 +1,58 @@
+package com.scottlogic.pod.spark.playground.steps;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.sql.SparkSession;
+import org.assertj.core.api.Assertions;
+
+import com.scottlogic.pod.spark.playground.WordCount;
+import com.scottlogic.pod.spark.playground.config.SparkIntegrationTestBase;
+
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class WordCountSteps extends SparkIntegrationTestBase {
+    private String path;
+    private WordCount wordCount;
+    private SparkSession spark;
+    private JavaSparkContext jsc;
+
+    @Before
+    public void before() {
+        this.spark = getSpark();
+        this.jsc = getJsc();
+    }
+
+    @Given("a new wordCount")
+    public void initiliseWordCount() {
+        this.wordCount = new WordCount(spark, jsc);
+    }
+
+    @When("I supply the path {string}")
+    public void setPath(String path) {
+        this.path = path;
+    }
+
+    @Then("I should get the expected count")
+    public void correctCountTest() {
```

Comment:
I'd be tempted to make this and `incorrectCountTest` paramertised to take a word count. Since the data is paramertised and it'd make them more reusable